adminservice/src/intTest/groovy/nu/yona/server/ActivityCategoriesTest.groovy

```diff
@@ -9,15 +9,11 @@ package nu.yona.server
 
 import groovy.json.*
 import nu.yona.server.test.AppService
-import spock.lang.Shared
 import spock.lang.Specification
 
 class ActivityCategoriesTest extends Specification
 {
-	@Shared
```

Comment:
Sorry, reverted, I did not know the exact meaning when I deleted it but found out that it also worked without :)